src/components/ProductContent/ProductMedia.tsx

```diff
@@ -1,16 +1,66 @@
 import React from 'react'
+import { useState, useEffect } from 'react'
+import cx from 'classnames'
+
+//TO DO: Retrieve images from database instead
```

Comment:
Normally, we don't add space here. You can simply select and change all occurrences.
```suggestion
//TODO: Retrieve images from database instead
```